registry/storage/blobstore.go

```diff
@@ -112,7 +112,7 @@ func (bs *blobStore) Enumerate(ctx context.Context, ingester func(dgst digest.Di
 		}
 
 		return ingester(digest)
-	})
+	}, driver.WalkWithStopOnErrSkipDir(true))
```

Comment:
I feel like this should be configurable _somehow_; once could argue that this should be the default option, I get that